### ISSUE TITLE
An example of sending a DeDust swap transaction in Cookbook

### ISSUE THREAD

**Issue by anton-trunov** [tact-lang/tact-docs#146]
Related issue: #149

Also, it would be nice to have an example of adding liquidity (as suggested by @Gusarich)

**Comment by novusnota** [comment]
Depends on #142

**Comment by a-bahdanau** [comment]
Я возьмусь
